### ISSUE TITLE
Batch Edit/Replace Content Across Multiple Email Templates

### ISSUE THREAD

**Issue by dshastry** [mjmlio/mjml-app#165]
Would it be possible to have a batch edit feature across multiple templates? For example, today I had a phone number I had to change across 17 emails and had to open each one to edit it.

Maybe something similar to brackets "Replace in Files" function?

![image](https://user-images.githubusercontent.com/4194561/29141483-08d343b6-7d14-11e7-8e95-730cd5827c02.png)

**Comment by meriadec** [comment]
Hey David, actually the replace is handled by the CodeMirror editor, which apply on the current opened buffer. As the projects are eventually not located to same place, I think this one can be confusing (where to put the UI, how to explain to user, etc.). I prefer to keep the app as simpler as possible, plus this is not really related to MJML but more a data/file problem.

For that specific need, perhaps you can use [mj-include](https://mjml.io/documentation/#mj-include), or a templating language where you put variables for phone number, etc. (check out [mustache](http://mustache.github.io/)).

TL;DR for the moment not implementing that.
